azure-pipelines.yml

```diff
@@ -0,0 +1,335 @@
+pool:
+  vmImage: 'ubuntu-latest'
+
+variables:
+  - group: sonarsource-build-variables
+
+resources:
+  repositories:
+    - repository: commonTemplates
+      type: git
+      name: pipelines-yaml-templates
+      ref:  refs/heads/feature/tycho
+
+stages:
+- template: stage-with-burgr-notifications.yml@commonTemplates
+  parameters:
+    burgrName: 'build'
+    burgrType: 'build'
+    stageName: 'build'
+    stageDisplayName: Build and stage to repox
+    jobs:
+    - job: build
+      displayName: Build and stage to repox
+      variables:
+        MAVEN_CACHE_FOLDER: $(Pipeline.Workspace)/.m2/repository
+        MAVEN_OPTS: '-Xmx3072m -Dmaven.repo.local=$(MAVEN_CACHE_FOLDER)'
+        commonMavenArguments: -B -e -Pdeploy-sonarsource -Dmaven.test.skip=true
+      steps:
+      - checkout: self
+        fetchDepth: 1
+      - task: CacheBeta@0
+        displayName: Cache Maven local repo
+        inputs:
+          key: maven | pom.xml
+          path: $(MAVEN_CACHE_FOLDER)
+      - task: DownloadSecureFile@1
+        displayName: 'Download Maven settings'
+        name: mavenSettings
+        inputs:
+          secureFile: 'maven-settings.xml'
+      - task: DownloadSecureFile@1
+        displayName: 'Download JKS keystore'
+        name: jks
+        inputs:
+          secureFile: 'SonarSource-2019-2021.jks'
+      - template: update-maven-version-steps.yml
+        parameters:
+          mavenSettingsFilePath: $(mavenSettings.secureFilePath)
+      - bash: |
+          set -e
+          sudo apt-get update
+          sudo apt-get install rng-tools
+          sudo service rng-tools start
+        displayName: Fix entropy to speed up JAR signing
+      - task: Maven@3
+        displayName: 'Run Maven deploy and sign'
+        condition: ne(variables['Build.Reason'], 'PullRequest')
```

Comment:
Look at the next task. The next one is executed on PR, and is the same Maven goal without signing. We don't want to sign PR artifacts, since they are not supposed to become official releases.

Reply:
Ah, so I guess the `displayName` should be updated accordingly :smile: to avoid confusion

Reply:
You mean adding "no sign"?